### PR TITLE
chore(testnet): always start the faucet

### PR DESCRIPTION
**Pull request by joshuef**
### Summary generated by Reviewpad on 25 Jul 23 19:17 UTC
This pull request updates the code related to the testnet and faucet in the project. It removes the option to launch the DBC faucet server and instead always starts it. It also includes some code refactoring and cleanup. After the changes, the testnet and faucet are launched successfully.

### CODE REVIEW

File: .github/workflows/memcheck.yml [hunk 1]
```diff
@@ -83,16 +83,8 @@ jobs:
...
-        run: cargo run --bin faucet --release -- claim-genesis
-        env:
-          SN_LOG: "all"
-        timeout-minutes: 2
-
       - name: Create and fund a wallet to pay for files storage
-        run: |
-          cargo run --bin faucet --release -- send 500 $(cargo run --bin safe --release -- wallet address | tail -n 1) | tail -n 1 > dbc_hex
-          cat dbc_hex | cargo run --bin safe --release -- wallet deposit --stdin
+        run: cargo run --bin safe --release -- wallet get-faucet
```
Comment (grumbach) [inline .github/workflows/memcheck.yml]: get faucet will only get 10DBC, is that enough?

**Review by grumbach — APPROVED** [review]
Wondering if we want a flag to disable the faucet if it's default now?

File: .github/workflows/memcheck.yml (hunk 1 above)
Comment (joshuef) [reply .github/workflows/memcheck.yml]: for now yeh, storage is 1 nano fixed price atm

**Comment by joshuef** [comment]
> Wondering if we want a flag to disable the faucet if it's default now?

I thought about that, but you can't do anything without it at the mo, so I left that off. 

I'm not against that if folk find it useful? (maybe follow up)

**Pull request merged**